Makes external airlocks and e-armory accessible to a lot more people

comma opened the pull request:
External airlocks now need either specific airlock access or solgov crew access.
E-armory needs either e-armory access or bridge access. The guns room needs speciifc access still.
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Sbotkin commented:
I have to remind that upon the SOP, the E-armory is only accessable by Line Officers during the Code Red and not accessable by anyone except CO, XO and SEA on other codes unless CO orders so. You should probably change the SOP then.

Hubblenaut commented:
Bridge armory includes the Liaison, SolGov Rep, Bridge Officers, Pilot, Pathfinder... and since the diplomat's quarters uses simple bridge access too, I guess you could say that any person you'd like to give access in there as Diplomat is going to have emergency armory access too.

When e-armory is its own access, why not give that access to the people that need it? Will cause less issues, including those that would come if people gain/lose bridge access for whatever reason.

comma commented:
It lets them enter the first part of e-armory, that contains highly sensitive equipment like grey spacesuits or PACMANs.
This will NOT get them into the part with guns.

hey-it-stev commented:
SOP will need to be edited accordingly

PurpleMartinJCK commented:
Doesn't everyone have SolGov crew access pretty much?

comma commented:
Yes.

comma committed:
Makes external airlocks accessible to all crew

Solgov crew, mind you.
You can get back inside the ship, but not go out randomly.

comma committed:
E-Armory only needs bridge access now

Except for the gunny part, that still needs strict E-Access. Other doors can be opened with either that or bridge access.
